datadog-checks-base/datadog_checks/checks/win/winpdh.py

```diff
@@ -7,13 +7,15 @@
 import win32pdh
 import _winreg
 
+DATA_TYPE_INT = win32pdh.PDH_FMT_LONG
+DATA_TYPE_DOUBLE = win32pdh.PDH_FMT_DOUBLE
 DATA_POINT_INTERVAL = 0.10
 SINGLE_INSTANCE_KEY = "__single_instance"
 class WinPDHCounter(object):
     # store the dictionary of pdh counter names
     pdh_counter_dict = {}
 
-    def __init__(self, class_name, counter_name, log, instance_name = None, machine_name = None):
+    def __init__(self, class_name, counter_name, log, instance_name = None, machine_name = None, precision=DATA_TYPE_DOUBLE):
```

Comment:
If we're going to default to `win32pdh.PDH_FMT_DOUBLE` when `precision = None`, I think it would feel more natural to me as far as the API goes if the default parameter for precision were `None`.